tabular/src/autogluon/tabular/models/fastainn/hyperparameters/searchspaces.py

```diff
@@ -35,3 +36,19 @@ def get_searchspace_multiclass(num_classes):
 
 def get_searchspace_regression():
     return get_searchspace_binary()
+
+
+def get_searchspace_quantile():
+    spaces = {
+        # See docs: https://docs.fast.ai/tabular.models.html
+        'layers': Categorical(None, [200, 100], [200], [500], [1000], [500, 200], [50, 25], [1000, 500],
+                              [200, 100, 50], [500, 200, 100], [1000, 500, 200]),
+        'emb_drop': Real(0.0, 0.2, default=0.0),
+        'ps': Real(0.0, 0.2, default=0.0),
+        'bs': Categorical(32, 64, 128, 512),
+        'lr': Real(1e-4, 1e-1, default=1e-2, log=True),
+        'epochs': Int(lower=5, upper=500, default=200),
+        'early.stopping.min_delta': 0.00001,
+        'early.stopping.patience': 50,
+    }
```

Comment:
You've verified the deviations from the default search space for other problem_types are beneficial for QUANTILE problem_type?

Reply:
This search space was used in our DQA project. Let me check this out once more.